examples/src/main/java/com/mapbox/navigation/examples/core/ReplayHistoryActivity.kt

```diff
@@ -171,6 +172,13 @@ class ReplayHistoryActivity : AppCompatActivity() {
             }
         })
 
+        mapboxNavigation.attachFasterRouteObserver(object : FasterRouteObserver {
```

Comment:
Showing all the routes helps debug faster route.

Also found that, if you reverse the route list - you'll get a faster route just about every time 🪂 
```
val reversed = alternatives.reversed()
navigationContext?.navigationMapboxMap?.drawRoutes(reversed)
navigationContext?.mapboxNavigation?.setRoutes(reversed)
```